CHANGELOG.md

```diff
@@ -285,7 +293,6 @@
 
 **Merged pull requests:**
 
-- Alternate fix for `env` variable in lib/config.js [\#23](https://github.com/zooniverse/panoptes-javascript-client/pull/23) ([shaunanoordin](https://github.com/shaunanoordin))
```

Comment:
Why was this removed?

Reply:
I'm not sure. I'm generating new versions of the changelog with a Ruby script that compiles the changes from the github API.